tools/src/converter.ts

```diff
@@ -13,6 +17,19 @@ function convertTypes(types: ITypeDescr[]): string[] {
     return Array.from(convertedTypes);
 }
 
+function expandTypes(types: ITypeDescr[], customTypes: ICustomType[]): ITypeDescr[] {
```

Comment:
Propose to make the `customTypes` a dictionary: `customTypes: Record<string, ICustomType>`